model_compression_toolkit/exporter/fully_quantized/keras/fully_quantized_quantize_wrapper.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2022 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from typing import Dict, Tuple, Any
+
+from keras.engine.base_layer import Layer
+from keras.layers import TFOpLambda
+from tensorflow import TensorShape
+from tensorflow_model_optimization.python.core.quantization.keras.quantize_config import QuantizeConfig
+from tensorflow_model_optimization.python.core.quantization.keras.quantize_wrapper import QuantizeWrapper
+
+
+class FullyQuantizedQuantizeWrapper(QuantizeWrapper):
+
+    """Quantizes the weights and activations of the keras layer it wraps, according
+    to the quantization config that is passed. This class was created to deal with TFOpLambda that could
+    not be used TF QuantizeWrapper since it does not implement compute_output_shape."""
+
+    def __init__(self,
+                 layer:Layer,
+                 quantize_config:QuantizeConfig,
+                 output_shape:Tuple[Any]=None,
+                 **kwargs:Dict[str, Any]):
+        """
+        Create a wrapper for a keras layer.
+
+        Args:
+            layer: The keras layer to be quantized.
+            quantize_config: `QuantizeConfig` to quantize the layer.
+            output_shape: The output shape of the layer.
+            **kwargs: Keyword arguments to build the base class QuantizeWrapper.
+        """
+
+        # TFOpLambda does not implement the method compute_output_shape which is mandatory for cloning the model
+        # and use TF transformations. For this reason, we add the output_shape in the layer configuration and
+        # add an implementation for compute_output_shape.
+        self._output_shape = output_shape
+        if isinstance(layer, TFOpLambda):
```

Comment:
Would it work for SlicingOpLambda as well? Because in MP we have the same issue with this layer type as well, and maybe we can solve it with this wrapper class.